PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -216,24 +216,39 @@
   selections = selections2016
 )
 
-from PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff import prefiringweight
+from PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff import prefiringweight
 #Next line will be updated once we get UL2016 maps
-run2_jme_2016.toModify( prefiringweight, DataEra = cms.string("2016BtoH"))
+run2_jme_2016.toModify( prefiringweight, DataEra = cms.string("2016BtoH"),  DataEraMuon = cms.string("2016"))
```

Comment:
better to redefine `DataEra` as `DataEraEcal`

Reply:
Done.